### PR TITLE
cli: Handle no npm info

### PR DESCRIPTION
**Pull request by ahrberg**
## Hey, I just made a Pull Request!

Solves the [problem](https://github.com/backstage/backstage/pull/3391#issuecomment-757926299) when no npm package info is found during `backstage-cli versions:bump`.

**How**
This PR handles empty response from `yarn info` used by `backstage-cli versions:bump`. Empty responses now throws a not found exception and is handled by ignoring the package bump and prints a message that the package is ignored due to package info not found. 
  
Note that all empty `stdout` from `yarn info` will be handled the same way regardless of `stderr`. Please feedback if this is ok or not.

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)

### CODE REVIEW

**Review by freben — APPROVED** [review]
Looks good! @Rugvip wanna have a final look and merge?

File: packages/cli/src/commands/versions/bump.ts [hunk 1]
```diff
@@ -55,7 +56,16 @@ export default async () => {
   // Track package versions that we want to remove from yarn.lock in order to trigger a bump
   const unlocked = Array<{ name: string; range: string; target: string }>();
   await workerThreads(16, dependencyMap.entries(), async ([name, pkgs]) => {
-    const target = await findTargetVersion(name);
+    let target: string;
+    try {
+      target = await findTargetVersion(name);
+    } catch (error) {
+      if (error instanceof NotFoundError) {
```
Comment (Rugvip) [inline packages/cli/src/commands/versions/bump.ts:63]: I prefer using `error.name === 'NotFoundError'` as a more resilient check

File: packages/cli/src/lib/versioning/packages.test.ts [hunk 2]
```diff
@@ -19,6 +19,7 @@ import path from 'path';
 import * as runObj from '../run';
 import { paths } from '../paths';
 import { fetchPackageInfo, mapDependencies } from './packages';
+import { NotFoundError } from '@backstage/backend-common';
```
Comment (Rugvip) [inline packages/cli/src/lib/versioning/packages.test.ts:22]:
```suggestion
import { NotFoundError } from '../errors';
```

**Review by Rugvip — COMMENTED** [review]
Looks good, thanks! 😁 

Two small nits then good to merge

**Comment by ahrberg** [comment]
Thanks for the feedback 👍  now updated

**Comment by freben** [comment]
@ahrberg The build error is due to a temporary breakage in master - please just rebase on latest master, and you should be good to go

**Review by freben — APPROVED** [review]

File: packages/cli/src/commands/versions/bump.ts (hunk 1 above)
Comment (freben) [reply packages/cli/src/commands/versions/bump.ts:63]: @ahrberg seems you still have this one to address

Comment (ahrberg) [reply packages/cli/src/commands/versions/bump.ts]: @freben sorry missed that one

**Comment by freben** [comment]
and now there's an unused import to remove :) ssoooo clooooose!

**Pull request merged**